components/cluster-inventory/api/v1beta1/cluster_types.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterSpec defines the desired state of Cluster
+type ClusterSpec struct {
+	Core            Core            `json:"core,omitempty"`
+	AdminKubeconfig AdminKubeconfig `json:"adminKubeconfig,omitempty"`
+	Status          ClusterStatus   `json:"status,omitempty"`
```

Comment:
What would be the purpose of this property?
It may be a confusing to have multiple statuses: [L58](https://github.com/kyma-project/control-plane/pull/3017/files#diff-ec32aeb8c1a993b252bf786796a7489fdee819a3b848ef06ead8a84c0ede3e3fR58), [L27](https://github.com/kyma-project/control-plane/pull/3017/files#diff-ec32aeb8c1a993b252bf786796a7489fdee819a3b848ef06ead8a84c0ede3e3fR27)